ingestion/src/metadata/ingestion/source/database/database_service.py

```diff
@@ -247,7 +248,16 @@ def get_status(self) -> SourceStatus:
     def get_services(self) -> Iterable[WorkflowSource]:
         yield self.config
 
-    def yield_database_service(self, config: WorkflowSource):
+    def yield_database_create_request_service(self, config: WorkflowSource):
```

Comment:
If we change this, should not we update the topology definition as well?

Reply:
Updated this method and the topology processor to `yield_create_request_database_service`